docker-compose/docker-compose.yml

```diff
@@ -162,6 +162,15 @@ services:
     ports:
       - 9200:9200
       - 9300:9300
+  kibana:
+    image: kibana:7.17.3
+    mem_limit: 1g
```

Comment:
for sure kibana doesn't need `1g`, `100m`?

Reply:
looks like things changed a lot since last time I played with it 😬 

helm charts defaults to 2g:
https://github.com/elastic/helm-charts/blob/main/kibana/values.yaml#L55

so if it plays well with 1g for you let's stick to that